routers/api/packages/maven/maven.go

```diff
@@ -140,9 +140,7 @@ func serveMavenMetadata(ctx *context.Context, params parameters) {
 	ctx.Resp.Header().Set("Content-Length", strconv.Itoa(len(xmlMetadataWithHeader)))
 	ctx.Resp.Header().Set("Content-Type", contentTypeXML)
 
-	if _, err := ctx.Resp.Write(xmlMetadataWithHeader); err != nil {
-		log.Error("write bytes failed: %v", err)
-	}
+	_, _ = ctx.Resp.Write(xmlMetadataWithHeader)
```

Comment:
Is there no other way to detect this apart from ignoring errors completely?

Reply:
The question is: why do you want to do so? What would you like to do when the "Write" fails .......... I do not see any useful case to handle such error.